source/includes/reference/_app_webhooks.md

```diff
@@ -0,0 +1,209 @@
+    "delivery": {
+        "version": "2.0.0",
+        "url": "https://your.webhook.url/12345" 
+    },
+    "trigger_on": "transfers#state-change", 
+    "scope": {
+        "domain": "application", 
+        "id": "<your client key>"
+    },
+    "created_by": {
+        "type": "application",
+        "id": "<your client id>", //clientId and key are not always the same
+    },
+    "created_at": "2008-09-15T15:53:00+00:00Z",
+}
+```
+
+### Request
+
+**`POST https://api.transferwise.com/v3/applications/{clientKey}/subscriptions`**
+
+`{clientKey}` can be received upon obtaining client credentials from our tech support.
+
+All fields listed below are required for creating a webhook subscription.
+
+Field                     | Description                                                             | Format
+---------                 | -------                                                                 | -----------
+name                      | A custom name for your webhook to ease with identification              | Text
+trigger_on                | `transfers#state-change`, `transfers#active-cases` or `balances#credit` | Text
+delivery.version          | The event representation semantic                                       | Text
+delivery.url              | Required. The URL where your server will be listening for events.       | Text
+
+
+### Response
+
+Field                     | Description                                                             | Format
+---------                 | -------                                                                 | -----------
+id                        | UUID that uniquely identifies the subscription                          | Text
+name                      | A custom name for your webhook to ease with identification              | Text
+trigger_on                | `transfers#state-change`, `transfers#active-cases` or `balances#credit` | Text
+delivery.version          | The event representation semantic                                       | Text
+delivery.url              | Required. The URL where your server will be listening for events.       | Text
+scope.domain              | Scope of this subscription, always "application" in this case           | Text
+scope.id                  | Client key used to create this subscription                             | Text
+created\_by.type          | Creator type. Always application in this case                           | Text
+created\_by.id            | Client id of the creator. Not always the same as the client key         | Text
+created\_at               | Timestamp of when the subscription was created                          | Text
+
+## Delete
+
+Deletes a subscription by its identifier.
+
+> Example Request:
+
+```shell
+curl -X DELETE "https://api.transferwise.com/v3/applications/{clientKey}/subscriptions/{id}" \
+  -H "Authorization: Bearer <your client level token>"
+```
+
+> Example Response:
+
+```json
```

Comment:
i thought we respond with 204 (NO_CONTENT) without a body

Reply:
Will test this